### PR TITLE
Run-4184/don't propagate multiple close events

### PR DESCRIPTION
**Pull request by pbaize**
Fixes an issue where close was being re-propagated multiple times.

[Win 10 tests](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5b48222054b21953031f3714)
[win 7 tests](https://testing-dashboard.openfin.co/#/app/sessions/api/completed/5b48212954b21953031f3713)

### CODE REVIEW

File: test/global_hotkey.tests.ts [hunk 1]
```diff
@@ -202,7 +202,7 @@ describe('GlobalHotkey', () => {
 
         GlobalHotkey.register(identity, hotkey, spy);
         //we simulate a window close.
-        ofEvents.emit(route.window('closed'), identity);
+        ofEvents.emit(route.window('closed', '*'), identity);
```
Comment (datamadic) [inline test/global_hotkey.tests.ts]: why did this need to change (just asking)?

Comment (pbaize) [reply test/global_hotkey.tests.ts]: The only place we were listening for `route.window('closed')` (with no subtopic/uuid/name) was the subscription manager, so I made it listen to `route.window('closed`, '*') which automatically get's re-emitted on any `route.window('closed', someId)`. That means to simulate a window close we needed to add the asterisk here to match.

File: src/browser/subscription_manager.ts [hunk 2]
```diff
@@ -28,7 +28,7 @@ export default class SubscriptionManager {
     constructor() {
         this.subscriptionList = new Map();
 
-        ofEvents.on(route.window('closed'), (identity: Identity) => {
+        ofEvents.on(route.window('closed', '*'), (identity: Identity) => {
```
Comment (pbaize) [inline src/browser/subscription_manager.ts]: @datamadic  here

File: test/global_hotkey.tests.ts (hunk 1 above)
Comment (datamadic) [reply test/global_hotkey.tests.ts]: also, looks like [this place](https://github.com/HadoukenIO/core/blob/develop/src/browser/external_window_event_adapter.js#L136) needs the `*` emit no? @pbaize

Comment (pbaize) [reply test/global_hotkey.tests.ts]: I don't think so, looks like the listener for that is here https://github.com/pbaize/core/blob/b1495d1065b9f31404576ab424add869f247e80a/src/browser/api/window.js#L508

Comment (datamadic) [reply test/global_hotkey.tests.ts]: 👍 you're right

**Pull request merged**